### PR TITLE
Update ChatApiSample to Swift 3 syntax

### PR DESCRIPTION
**Pull request by baz8080**
* Updates Pod dependencies to Swift 3 compatible versions
* Updates Chat SDK to latest version

@divo please review

### CODE REVIEW

File: ChatAPISampleApp/ZDCChatAPI Sample App/ChatViewController.swift [hunk 1]
```diff
@@ -76,21 +76,21 @@ class ChatViewController: UIViewController {
 
 extension ChatViewController {
   
-  @IBAction func pickImage(sender: AnyObject) {
+  @IBAction func pickImage(_ sender: AnyObject) {
     let pickerController = DKImagePickerController()
     
-    pickerController.singleSelect = true
+    pickerController.singleSelect = false
```
Comment (baz8080) [inline ChatAPISampleApp/ZDCChatAPI Sample App/ChatViewController.swift]: Set back to true

File: ChatAPISampleApp/Podfile [hunk 2]
```diff
@@ -6,8 +6,8 @@ use_frameworks!
 target 'ZDCChatAPI Sample App' do
   pod 'SDWebImage', '~>3.8'
   pod 'DKImagePickerController'
-  pod 'NSDate+TimeAgo'
-  pod 'ZDCChat/API', '1.3.0.1'
-  pod 'JLToast', '~> 1.4'
+  pod 'DateToolsSwift'
```
Comment (divo) [inline ChatAPISampleApp/Podfile]: I know they weren't there but would you mind adding version numbers to these to lock them in?

pod 'DateToolsSwift' ~> '2.0.3'
pod 'DKImagePickerController' ~> '3.5.7'

Are the current versions for each.

Comment (baz8080) [inline ChatAPISampleApp/Podfile]: Will do

File: ChatAPISampleApp/ZDCChatAPI Sample App/ChatViewController.swift (hunk 1 above)
Comment (baz8080) [inline ChatAPISampleApp/ZDCChatAPI Sample App/ChatViewController.swift]: My bad, I was testing multiple attachment uploads, I will reset it.

**Pull request merged**